### PR TITLE
Fix set artwork when playing from webview

### PR DESCRIPTION
**Pull request by kesha-antonov**
#### What's this PR does?

Fix for missing artwork.
When playing from webview apparently it resets controls and artwork too. 

#### Which issue(s) is it related to?

https://github.com/tanguyantoine/react-native-music-control/issues/208

### CODE REVIEW

**Comment by kesha-antonov** [comment]
https://drive.google.com/open?id=1523d_v3cof6RWpQJWyhJDuCH5ax8oFPj

File: ios/MusicControlManager.m [hunk 1]
```diff
@@ -285,54 +282,66 @@ - (void)sendEventWithValue:(NSString*)event withValue:(NSString*)value{
...
-                    if ([artworkUrl isEqual:self.artworkUrl]) {
-                        MPNowPlayingInfoCenter *center = [MPNowPlayingInfoCenter defaultCenter];
-                        MPMediaItemArtwork *artwork = [[MPMediaItemArtwork alloc] initWithImage: image];
-                        NSMutableDictionary *mediaDict = (center.nowPlayingInfo != nil) ? [[NSMutableDictionary alloc] initWithDictionary: center.nowPlayingInfo] : [NSMutableDictionary dictionary];
-                        [mediaDict setValue:artwork forKey:MPMediaItemPropertyArtwork];
-                        center.nowPlayingInfo = mediaDict;
-                    }
-                });
+            // Check if URL wasn't changed in the meantime
+            if (![artworkUrl isEqual:self.artworkUrl]) {
```
Comment (tanguyantoine) [inline ios/MusicControlManager.m]: I think you changed the logic here.

Comment (tanguyantoine) [reply ios/MusicControlManager.m]: nevermind

**Comment by kesha-antonov** [comment]
@tanguyantoine Nice!

**Pull request merged**